Apsitvarkom.Api/Controllers/PollutedLocationController.cs

```diff
@@ -71,7 +84,41 @@ public async Task<ActionResult<PollutedLocationGetResponse>> GetById(string id)
         try
         {
             var location = await _repository.GetByPropertyAsync(x => x.Id.ToString() == id);
-            return location is not null ? Ok(location) : NotFound($"Polluted location with the specified id '{id}' was not found.");
+
+            if (location is null) return NotFound($"Polluted location with the specified id '{id}' was not found.");
+
+            var mappedLocation = _mapper.Map<PollutedLocationGetResponse>(location);
+            if (mappedLocation is null) return StatusCode(StatusCodes.Status500InternalServerError);
+
+            return Ok(mappedLocation);
+        }
+        catch (Exception)
+        {
+            return StatusCode(StatusCodes.Status500InternalServerError);
+        }
+    }
+
+    [ProducesResponseType(StatusCodes.Status201Created)]
+    [ProducesResponseType(StatusCodes.Status400BadRequest, Type = typeof(List<string>))]
+    [ProducesResponseType(StatusCodes.Status500InternalServerError)]
+    [HttpPost("Create")]
+    public async Task<ActionResult<PollutedLocationGetResponse>> Create(PollutedLocationCreateRequest pollutedLocationCreateRequest)
```

Comment:
This comment makes more sense, since `PollutedLocationResponse` has somewhat the same meaning as `PollutedLocationGetResponse`. I use `Get` here not as in a HTTP GET, but more as a "Retrieve"